### DISCUSSION TITLE
Conductor Server Authentication & Authorization (OAuth/OAuth2.0)

### DISCUSSION THREAD

**Discussion by maheshyaddanapudi** [Netflix/conductor#2189] [Ideas]
OAuth2.0 Role Based Security! - Spring Security with easy configuration to secure the Conductor server APIs.

- With default role definitions
- With customization flexibility
- Optional : Enabling / Disabling security
- Compatible with most of the OAuth2.0 providers

Raised a PR [Pull Request](https://github.com/Netflix/conductor/pull/2200)

**Comment by aravindanr** [comment]
@maheshyaddanapudi Thanks for the contribution. I posted a [comment](https://github.com/Netflix/conductor/pull/2200#issuecomment-827201836) in the PR. In general, we feel that authorization can be bundled as part of a fork.
